html_css/js/slider.js

```diff
@@ -0,0 +1,138 @@
+export default class Slider {
+
+  constructor(selector, data) {
+    this.selector = selector;
+    this.data = data;
+  }
+
+  initSlider() {
+    this.slider = document.querySelector(this.selector);
+    this.processDataForSlides(this.data);
+    this.initIventListener();
+    this.page = 0;
+    this.buttonHandler(this.page);
+    this.changeSlidesQuantity()
+  }
+
+  renderButtons(buttonSide) {
+    this.button = document.createElement('button');
+    this.button.classList.add('slider__arrow-btn');
+
+    this.button.innerHTML = this.getButtonTemplate(buttonSide);
+
+    this.slider.appendChild(this.button);
+  }
+
+  getButtonTemplate(buttonSide) {
+    return `<svg class="slider__arrow arrow-${buttonSide}" viewBox="0 0 33 32" width="40">
+      <use class="arrow-${buttonSide}" href="./assets/images/sprite.svg#icon-slide-${buttonSide}"></use>
+      </svg>`;
+  }
+
+  initIventListener() {
+    this.slider.addEventListener('click', (event) => {
+      this.isButtonClass(event) ? this.clickHandler(event) : null;
```

Comment:
Better to replace by simple if statement

Reply:
Done